scripts/add_rtp_process_record.py

```diff
@@ -6,15 +6,13 @@
 """Add individual processing record to M&C database from RTP."""
 
 import warnings
+import importlib
 from pkg_resources import parse_version
 
 import numpy as np
 from astropy.time import Time
 
 import hera_mc.mc as mc
-import hera_qm
-import hera_opm
```

Comment:
Unfortunately, hera_opm is not on setuptools-scm yet, so this will still break.

Reply:
we've merged a new hera_opm that uses setuptools scm